lib/ddp-client.js

```diff
@@ -146,6 +146,8 @@ DDPClient.prototype._message = function(data) {
 
   } else if (data.msg === "connected") {
     self.session = data.session;
+    self._isConnecting = false;
+    self._isReconnecting = false;
     self.emit("connected");
```

Comment:
This emit triggers a call to this listener (when registered by providing a `connect` function):
https://github.com/oortcloud/node-ddp-client/blob/master/lib/ddp-client.js#L314-L320

In the current code, that listener function calls `connect` (passing `self._isReconnecting` as a parm)  *before* clearing `self._isReconnecting` in the subsequent lines. With this change, the value of `self._isReconnecting` will be cleared prior to that call being made, potentially changing the behavior of a registered `connect` function.